<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_produto</class>
 <widget class="QMainWindow" name="tela_produto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>392</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>30</pointsize>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>Cadastrar Produto</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/newPrefix/back.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>Cadastrar Produto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>ID do Produto:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>Nome do Produto:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>Valor do Produto:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="produto_id">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="produto_nome">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="produto_valor">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>230</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="botao_cadastrar_produto">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>Cadastrar Produto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_voltar_login">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../background/teste.qrc"/>
 </resources>
 <connections/>
</ui>
